### ISSUE TITLE
version 1.3.11 composer install fail with laravel 5.1

### ISSUE THREAD

**Issue by gan068** [intrip/laravel-authentication-acl#126]
Could not scan for classes inside "acl\vendor/jacopo/authenticati
  on-sentry/src/migrations" which does not appear to be a file nor a folder

**Comment by intrip** [comment]
It's fixed in 1.3.15

**Comment by gan068** [comment]
Laravel 5.1 LTS can not using version 1.3.15

**Comment by ghost** [comment]
Just a note that 1.3.11 installation similarly fails on a fresh Laravel 5.0.33 instance. After adding 1.3.11 as a requirement in `composer.json`, running `composer update` yields:

```
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 7 installs, 0 updates, 0 removals
  - Installing gregwar/captcha (v1.0.11): Downloading (100%)         
  - Installing jacopo/authentication-sentry (1.0.1): Downloading (100%)         
  - Installing psr/http-message (1.0.1): Downloading (100%)         
  - Installing guzzlehttp/psr7 (1.4.2): Downloading (100%)         
  - Installing intervention/image (2.4.1): Downloading (100%)         
  - Installing illuminate/html (v5.0.0): Downloading (100%)         
  - Installing jacopo/laravel-authentication-acl (1.3.11): Downloading (100%)         
jacopo/authentication-sentry suggests installing happydemon/txt (Required Text helpers when using the Kohana implementation)
intervention/image suggests installing ext-imagick (to use Imagick based image processing.)
intervention/image suggests installing intervention/imagecache (Caching extension for the Intervention Image library)
Package illuminate/html is abandoned, you should avoid using it. Use laravelcollective/html instead.
Writing lock file
Generating autoload files
Warning: Ambiguous class resolution, "CreateUsersTable" was found in both "$baseDir . '/database/migrations/2014_10_12_000000_create_users_table.php" and "/home/<user>/public_html/vendor/jacopo/laravel-authentication-acl/database/migrations/2014_02_19_095545_create_users_table.php", the first will be used.
Warning: Ambiguous class resolution, "CreatePasswordResetsTable" was found in both "$baseDir . '/database/migrations/2014_10_12_100000_create_password_resets_table.php" and "/home/<user>/public_html/vendor/jacopo/laravel-authentication-acl/database/migrations/2014_10_12_100000_create_password_resets_table.php", the first will be used.

                                                                                                                                                                        
  [RuntimeException]                                                                                                                                                    
  Could not scan for classes inside "/home/<user>/public_html/vendor/jacopo/authentication-sentry/src/migrations" which does not appear to be a file nor a folder                                                                                                                                                                  
                                                                                                                                                                        

update [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--lock] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [--with-dependencies] [--with-all-dependencies] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [-i|--interactive] [--root-reqs] [--] [<packages>]...

```

(`composer install` yields the same error, IIRC.)

**Comment by ghost** [comment]
Therefore:

- this issue should be reopened and fixed in the 1.3.11 branch (which seems unlikely, given #164); or
- a later branch should be modified to work with Laravel 5.0 and 5.1, and the documentation updated accordingly; or
- support for Laravel 5.1 and earlier should be explicitly dropped. I.e. instead of saying, somewhat misleadingly, "for Laravel 5.1/5.0 use the version 1.3.11," readme.md should say, "Laravel 5.1 and earlier are not supported." Likewise https://github.com/intrip/laravel-authentication-acl/blob/1.4/docs/index.md should not give instructions to install 1.3.11 onto 5.1 or 5.0.

**Comment by ghost** [comment]
A workaround is to run `mkdir /home/<user>/public_html/vendor/jacopo/authentication-sentry/src/migrations` and then re-run `composer install`.
